cli/README.md

```diff
@@ -57,6 +57,110 @@ yo mern:component Sidebar --stateless
 
 Produces `app/components/Sidebar.js`.
 
+### CRUD
+
+Generates a series of files used in a basic CRUD scaffolding.
+
+Example:
+
+```bash
+yo mern:crud Car
+```
+
+Produces:
+
+`api/car/car.service.js`
+
+`api/car/car.controller.js`
+
+`api/car/index.js`
+
+`app/actions/carActions.js`
+
+`app/reducers/carReducer.js`
+
+`app/services/carService.js`
+
+`app/components/car/CarPage.js`
+
+`app/components/car/CarList.js`
+
+`app/components/car/Car.js`
+
+`app/components/car/CarForm.js`
+
+`app/components/car/CarAddPage.js`
+
+`app/components/car/CarEditPage.js`
+
+But wait: that's not all. You still need to do a couple of minor changes/additions.
+
+#### api/server.js
+
+`app.use(require('./car/index.js'));`
+
+#### app/actions/actionTypes.js
+
+`export const LOAD_CAR_SUCCESS = 'LOAD_CAR_SUCCESS';`
+
+`export const GET_CAR_SUCCESS = 'GET_CAR_SUCCESS';`
+
+`export const CREATE_CAR_SUCCESS = 'CREATE_CAR_SUCCESS';`
+
+`export const UPDATE_CAR_SUCCESS = 'UPDATE_CAR_SUCCESS';`
+
+`export const DELETE_CAR_SUCCESS = 'DELETE_CAR_SUCCESS';`
+
+#### app/services/apiEndpoints.js
+
+`export const GET_CARS = '/cars';`
+
+`export const GET_CAR = '/cars';`
+
+`export const POST_CAR = '/cars';`
+
+`export const PUT_CAR = '/cars';`
+
+`export const DELETE_CAR = '/cars';`
+
+#### app/reducers/index.js
+
+`import { cars, car } from './carReducer';`
+
+```
```

Comment:
Tip: you may use an indicator for the language so that GitHub does syntax highlighting:

    ```javascript
    // your code
    ```

Would produce:

```javascript
// your code
```